### ISSUE TITLE
Option type "number" already registered in /.../wp-content/plugins/unyson/framework/core/components/backend.php on line 257

### ISSUE THREAD

**Issue by johnkuypers** [ThemeFuse/Unyson#3965]
Hi yo i got a problem since a few day i updated php and wp to all latest but no ok i got error:
 Warning: Option type "number" already registered in /.../wp-content/plugins/unyson/framework/core/components/backend.php on line 257

Thanks

**Comment by Lammbock13** [comment]
*Same problem here :/

**Comment by johnkuypers** [comment]
That is strange I just payed for my annually fee do you 

Is there some scam here ?? My customer was 3 weeks online!!!!! I think we have a serous problem here!

Sie haben eine Zahlung über $59,00 USD an Paddle.com <http://paddle.com/> Market Limited gesendet (help@paddle.com <mailto:help@paddle.com>).

**Comment by johnkuypers** [comment]
And you told me you updated the theme this isn’t correct if I download it envato.com the same problems appears!

**Comment by zunamidev** [comment]
Hey, this is not from the framework just return the error, if you want I can try to fix it, just write to me.

**Comment by zunamidev** [comment]
Its an issue from your Theme

**Comment by johnkuypers** [comment]
Hi i am sorry maybe you are right! but anyway thank you for reacting now! i wrote the error on 9 may and if you do a google search  it all points to Unyson and i thought themeFuse is the builder of that

**Comment by zunamidev** [comment]
Yes, it points to unyson because its the framework on which the theme is built on. If fixed it for @Lammbock13 in his theme file there was a doubled registered numbers type.

**Comment by zunamidev** [comment]
I was first a little bit confused but in the documentation stand it that you can register your own types and this is handled by the theme

**Comment by kaoskurrent** [comment]
Hi, I seem to be having the same issue on a site I'm working on but I can't find where in the theme the number type is registered twice. Where were you able to find the double registration? 

Thanks in advance?

**Comment by zunamidev** [comment]
I founded it just once registered but if you remove it the bug is fixed and nothing changes on the theme site. I'm searching for more and where it is registered. You can try to remove the first registration.

**Comment by kaoskurrent** [comment]
Thanks, I would appreciate any help! I looked in the main theme and child theme's functions.php files where the unyson documentation says the registration goes but found nothing.

**Comment by zunamidev** [comment]
When you need something you can write me a mail I can take a look
